Slack&whatsapp-notifier.py

```diff
@@ -94,72 +85,60 @@ def execute_query(mycursor, query, t):
     result = mycursor.fetchall()[0][0]
     return result
 
-# Checking if failed transactions are more than threshold
-def process_failed_transactions(result,y):
-    if result >= y:
-        print("ERROR CONFIRMED")
-        send_failed_transaction_notifications()
+def if_threshold_reached(result, threshold):
+    if result >= threshold:
+        logging.error("ERROR CONFIRMED")
+        send_notifications()

```

Comment:
if_threshold_reached name suggest that the function is only checking the threshold, it should not send notification

Reply:
if_threshold_reached() is the function which checks for threshold and then does a function if threshold is reached, i.e send_notifications()